app/main.development.js

```diff
@@ -108,7 +108,27 @@ const installExtensions = async () => {
   }
 };
 
-const { ipcMain } = require("electron");
+const { ipcMain, systemPreferences } = require("electron");
+
+const changeDarkMode = (newAppearance) => {
```

Comment:
Small naming nit: this should be `changeTheme`, `selectTheme`, `selectAppearance` or something like that, given that it's not changing the dark mode in any way, but rather selecting whether to use dark or light mode.

You could also just pass the `theme-*` value directly instead of performing another test.

Reply:
I will remove this part since it's not used anywhere else and integrate the key line into the code handling the startup mode below